### PR TITLE
lectures list

### PR DESCRIPTION
**Pull request by YanaYahello**

### CODE REVIEW

File: homeworks/react-basics/yahello_yana/kottans-lectures/public/data.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
```
Comment (KinjinZzz) [inline homeworks/react-basics/yahello_yana/kottans-lectures/public/data.json]: You don't need this file, you can get data from ```https://raw.githubusercontent.com/kottans/chernivtsi-frontend-2019/master/homeworks/react-basics/data.json```

File: homeworks/react-basics/yahello_yana/kottans-lectures/src/lecturesDetails.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        super(props);
+        this.state = {isOpened: false};
+    }
+
+    toggleState() {
+        this.setState({isOpened: !this.state.isOpened})
+    }
+
+    render() {
+        let isShow;
```
Comment (KinjinZzz) [inline homeworks/react-basics/yahello_yana/kottans-lectures/src/lecturesDetails.js]: You don't need `isShow` here. You can just check if `this.state.isOpened ` is `true` in return, this should look like: 
```
{this.state.isOpened && <div>...</div>}
```

**Comment by KinjinZzz** [comment]
Please move files from `homeworks/react-basics/yahello_yana/kottans-lectures/` folder to `homeworks/react-basics/YanaYahello/`, because you don't need extra homework folder here.

File: homeworks/react-basics/yahello_yana/kottans-lectures/public/data.json (hunk 1 above)
Comment (YanaYahello) [reply homeworks/react-basics/yahello_yana/kottans-lectures/public/data.json]: Thank you, fixed!

File: homeworks/react-basics/yahello_yana/kottans-lectures/src/lecturesDetails.js (hunk 2 above)
Comment (YanaYahello) [reply homeworks/react-basics/yahello_yana/kottans-lectures/src/lecturesDetails.js]: fixed

**Comment by YanaYahello** [comment]
> Please move files from `homeworks/react-basics/yahello_yana/kottans-lectures/` folder to `homeworks/react-basics/YanaYahello/`, because you don't need extra homework folder here.

Could you check, please, if I clearly understood you according this comment. Because, I have folder with name YanaYahello, but with css task.

**Comment by KinjinZzz** [comment]
> > Please move files from `homeworks/react-basics/yahello_yana/kottans-lectures/` folder to `homeworks/react-basics/YanaYahello/`, because you don't need extra homework folder here.
> 
> Could you check, please, if I clearly understood you according this comment. Because, I have folder with name YanaYahello, but with css task.

Almost, every homework folder need to have same name, so rename it from `yahello_yana` to `YanaYahello` (as in css task)

**Comment by YanaYahello** [comment]
> > > Please move files from `homeworks/react-basics/yahello_yana/kottans-lectures/` folder to `homeworks/react-basics/YanaYahello/`, because you don't need extra homework folder here.
> > 
> > 
> > Could you check, please, if I clearly understood you according this comment. Because, I have folder with name YanaYahello, but with css task.
> 
> Almost, every homework folder need to have same name, so rename it from `yahello_yana` to `YanaYahello` (as in css task)

thanks, done!

**Comment by KinjinZzz** [comment]
Thank you 👍

**Pull request merged**